src/pageObject.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  };
 };
 
-class PageObject {
+class PageObject extends Header {
```

Comment:
`PageObject` shouldn't extend `Header`, there is no such requirements in task description, you shouldn't change it